assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -21,43 +21,54 @@ import { __ } from '@wordpress/i18n';
 /**
  * External dependencies
  */
-import { useContext, useEffect } from 'react';
+import { useCallback, useContext, useEffect } from 'react';
 
 /**
  * Internal dependencies
  */
 import { ApiContext } from '../../api/apiProvider';
+import { useConfig } from '../../config';
 import GoogleAnalyticsSettings from './googleAnalytics';
-import PublisherLogoSettings from './publisherLogo';
 import { Wrapper, Header, Heading, Main } from './components';
 
 function EditorSettings() {
+  const { capabilities: { canManageSettings } = {} } = useConfig();
+
   const {
     actions: {
       settingsApi: { fetchSettings, updateSettings },
     },
     state: {
-      settings: { googleAnalyticsId, publisherLogos },
+      settings: { googleAnalyticsId },
     },
   } = useContext(ApiContext);
 
   useEffect(() => {
     fetchSettings();
   }, [fetchSettings]);
 
+  const handleCompleteUpdateId = useCallback(
+    ({ newGoogleAnalyticsId }) => {
+      const updatedSettings = {
+        googleAnalyticsId:
+          typeof newGoogleAnalyticsId === 'string' && newGoogleAnalyticsId,
+      };
```

Comment:
Am checking for a string here to know that it's the analytics ID that is getting updated so that the same handler can be used for publisher logos.

Reply:
Would a publisher logo be a blob at this point or a string of a path?

Reply:
it's a blob

Reply:
and it'll come in as it's own key in the prop object called 'publisherLogo'. Here i'm really checking to see if the update to the ga id is either a) a new id or b) removing an existing id or c) nothing to do with the ga id at all (which would mean it's false)